src/Microsoft.AspNetCore.WebSockets.Protocol/Properties/AssemblyInfo.cs

```diff
@@ -1,37 +1,13 @@
-using System.Reflection;
+﻿using System.Reflection;
 using System.Resources;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-// General Information about an assembly is controlled through the following 
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
 [assembly: AssemblyTitle("Microsoft.Net.WebSockets")]
-[assembly: AssemblyDescription("")]
```

Comment:
This one prevented the compiler from using the description field from project.json.